### ISSUE TITLE
Add support for using custom TLS certificates e.g. letsencrypt

### ISSUE THREAD

**Issue by x-ji** [cogini/elixir-deploy-template#11]
For any real production deployment, TLS should be indispensable?

**Comment by x-ji** [comment]
Or do you tend to put the TLS at the CDN level instead of directly on the Phoenix app etc.

**Comment by reachfh** [comment]
Yes, I think certs are a requirement for most websites. I plan to add letsencrypt support in the next iteration. I tend to use AWS support in the Application Load Balancer for larger applications. That's another thing I have in the pipeline....
